soh/soh/ResourceManagerHelpers.cpp

```diff
@@ -395,49 +404,49 @@ extern "C" char* ResourceMgr_LoadArrayByNameAsVec3s(const char* path) {
     //     return (char*)res->CachedGameAsset;
     // else
     // {
-        Vec3s* data = (Vec3s*)malloc(sizeof(Vec3s) * res->Scalars.size());
+    Vec3s* data = (Vec3s*)malloc(sizeof(Vec3s) * res->Scalars.size());
 
-        for (size_t i = 0; i < res->Scalars.size(); i += 3) {
-            data[(i / 3)].x = res->Scalars[i + 0].s16;
-            data[(i / 3)].y = res->Scalars[i + 1].s16;
-            data[(i / 3)].z = res->Scalars[i + 2].s16;
-        }
+    for (size_t i = 0; i < res->Scalars.size(); i += 3) {
+        data[(i / 3)].x = res->Scalars[i + 0].s16;
+        data[(i / 3)].y = res->Scalars[i + 1].s16;
+        data[(i / 3)].z = res->Scalars[i + 2].s16;
+    }
 
-        // res->CachedGameAsset = data;
+    // res->CachedGameAsset = data;
 
-        return (char*)data;
+    return (char*)data;
     // }
 }
 
 extern "C" CollisionHeader* ResourceMgr_LoadColByName(const char* path) {
-    return (CollisionHeader*) ResourceGetDataByName(path);
+    return (CollisionHeader*)ResourceGetDataByName(path);
```

Comment:
seeing all of these whitespace changes just reminds me of how we need to get clang-format into ship

Reply:
LOL. This only happened because I cut and pasted a couple of times to get IntelliSense to fix itself XD